src/PiCraft.jl

```diff
@@ -83,9 +86,9 @@ module player
     function getPos()
         res = PiCraft.mc_send("player.getPos()", true)
         @assert length(res) == 3
-        return (parse(Float64, res[1]),
+        return [parse(Float64, res[1]),
                 parse(Float64, res[2]),
-                parse(Float64, res[3]))
+                parse(Float64, res[3])]
     end
```

Comment:
Tuple is immutable making it less flexible if user wants to deal with relative coordinates.
For example:
`p = player.Pos()`
`p += [15, 1, 0]`
`world.setBlock(p[1], p[2], p[3])`

Reply:
I want to think a little bit about this one. Can you make a separate PR for it please, so that I can merge all the other changes? They are all good.